database/migrations/2018_07_16_173334_CreateTeamMembersTable.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTeamMembersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('team_members', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('rollNo')->unsigned();
```

Comment:
If this is unique for app_users, then this can be a foreign key. If this is the case then put a foreign key constraint.
`$table->foreign('rollNo')->references('rollNo')->on('app_users');`

I think `string` can't be `unsigned`.